setup.py

```diff
@@ -69,6 +69,7 @@
         'six>=1.9, <2.0',
         'toml>=0.9, <1.0',
         'sseclient==0.0.14',
+        'retrying',
```

Comment:
We have two `setup.py` and `requirements.txt` pair of files (one for each package, dcos and dcoscli), this one is for dcos, where that dependency is not used. I think you want to revert this line.

Reply:
thank you sir!  will do!